pkg/controller/route/routecontroller.go

```diff
@@ -79,7 +84,12 @@ func (rc *RouteController) reconcile(nodes []api.Node, routes []*cloudprovider.R
 	for _, route := range routes {
 		routeMap[route.TargetInstance] = route
 	}
+
+	maxRetries := 5
+	maxConcurrentRouteCreations := 200
```

Comment:
Actually, wait. This constant needs to be queried through the cloudprovider, e.g. AWS may not have a bounded concurrency issue. I don't object to fixing it at this level, but it needs a little more work.

We can handle that in a follow-on PR if you want, though, but please file an issue.